.gitignore

```diff
@@ -28,3 +28,5 @@ npm_licenses.tar.bz2
 /.build
 
 /**/node_modules
+
+.idea
```

Comment:
You can add your this in your gitignore, or propose this in another PR - let's keep things separate (: